services/horizon/internal/ingest/ingestion.go

```diff
@@ -226,6 +258,7 @@ func (ingest *Ingestion) Operation(
 // `history_operation_participants` table.
 func (ingest *Ingestion) OperationParticipants(op int64, aids []xdr.AccountId) {
 	for _, aid := range aids {
+		go ingest.waitAndPublish(ingest.getCommitChannel(), aid.Address())
```

Comment:
no mention of this in the function docs or code comment. also repeated in other places in the pr